src/components/colorPicker/ColorPickerDialog.tsx

```diff
@@ -197,7 +225,7 @@ class ColorPickerDialog extends PureComponent {
   };
 
   renderHeader() {
-    const {useCustomTheme, accessibilityLabels} = this.props;
+    const {buttonColor, accessibilityLabels} = this.props;
```

Comment:
`buttonColor` is too vague.. what about `confirmButtonColor`?

Reply:
Ok. Changed to 'doneButtonColor'.